packages/trace-viewer/src/ui/errorsTab.tsx

```diff
@@ -172,7 +100,7 @@ function Error({ message, error, errorId, sdkLanguage, pageSnapshot, revealInSou
   </div>;
 }
 
-function FixWithAIButton({ conversationId, value, onChange, error, diff, pageSnapshot }: { conversationId: string, value: boolean, onChange: React.Dispatch<React.SetStateAction<boolean>>, error: string, diff?: string, pageSnapshot?: string }) {
+function FixWithAIButton({ conversationId, value, onChange, prompt }: { conversationId: string, value: boolean, onChange: React.Dispatch<React.SetStateAction<boolean>>, prompt: string }) {
```

Comment:
I wonder if we should remove those to focus on the important button for now.

Reply:
i don't think it'll need any more tending, so let's keep it for now